js/src/personal/kazi/Kazi.module.css

```diff
@@ -7,4 +7,21 @@
   @media (max-width: $mantine-breakpoint-md) {
     font-size: rem(50px);
   }
-}
\ No newline at end of file
+}
+
+html {
```

Comment:
nit: is this required to be on html tag? Should try to scope things as tightly as possible

Reply:
So I want it to be able to snap to each of the sections when you scroll and I tried it by specifying sections at first and it didn't work.